android/src/app/organicmaps/widget/placepage/PlacePageView.java

```diff
@@ -784,6 +785,12 @@ public void onCompassUpdated(double north)
     }
   }
 
+  @Override
+  public void onLocationDisabled() {}
+
+  @Override
+  public void onLocationResolutionRequired(@NonNull PendingIntent pendingIntent) {}
```

Comment:
Should it be handled somewhere else? Is this "event" really needed by every location observer?

Reply:
It may lead to confusion: e.g. every observer may implement an attempt to resolve permission. Can we "hide" it inside the implementation and ask permission request without notifying anyone else?